src/amend/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Mutate given list of patter strings
+ * @param {string[]} list
+ * @returns {string[]}
+ */
+export function amend (list) {
+  try {
+    // Risk: Uses lookbehind assertion, avoid breakage in parsing by using RegExp constructor
+    new RegExp('(?<! cu)bot').test('dangerbot') // eslint-disable-line prefer-regex-literals
+    // Addresses: Cubot device
+    list.splice(list.lastIndexOf('bot'), 1)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/omrilotan/isbot/pullRequest?prid=7728710)
```suggestion
    list.splice(list.lastIndexOf("bot"), 1)
```